### DISCUSSION TITLE
blog/tech/010/

### DISCUSSION THREAD

**Discussion by giscus** [Vanadiry/blog-comments#22] [blog-comments]
# blog/tech/010/

你可能是不小心删了 Authenticator 而且没备份，也可能是扫 MFA 二维码时候手太快给点掉了，总之现在是被两步验证给卡了，登录不上 E5 管理员账号，而且你登陆不上去的这个管理员账号是唯一的管理员… 免费的开发人员即时沙盒前段时间好像不能注册了，会提示没有资格。按照这个文章的步骤来走，应该可以帮你重新拿到管理员账户。全程只需要电话联系即可，不需要提供截图等证明。 须知只要这个账号是你用

https://magic.vanadiry.com/blog/tech/010/

**Comment by moreant** [comment]
我也遇到一样的问题了，兜兜转转联系上了人工客服，基本是一样的步骤，坐等明天技术回拨了。
就是工单怎么进 outlook 的垃圾邮件了，我 ban 我自己？🤣

**Reply by Vanadiry** [reply]
祝成功！

印象里我收到的被归类到“其他”里面了∠( ᐛ 」∠)＿

**Reply by moreant** [reply]
成功了~

周二联系我确定信息，周五下午就通知我清掉二步验证了。

**Reply by Vanadiry** [reply]
好耶！恭喜🎉

**Comment by DataEraserC** [comment]
最近尝试找回丢失已久的全局管理员账号，发现微软客服电话打过去是大语言模型处理
